ethdev/ethdev.go

```diff
@@ -350,9 +363,16 @@ func OptRxMode(conf RxMode) Option {
 			split_hdr_size: C.ushort(conf.SplitHdrSize),
 			offloads:       C.ulong(conf.Offloads),
 		}
+		setRxPktLen(c, conf.MTU)
 	}}
 }
 
+func setRxPktLen(c *ethConf, n uint32) {
+	rxptr := unsafe.Pointer((*C.struct_rte_eth_rxmode)(&c.conf.rxmode))
+	p := unsafe.Pointer(uintptr(rxptr) + C.RX_MODE_LEN_OFF)
+	*(*C.uint)(p) = (C.uint)(n)
```

Comment:
Well that's risky. Using `uint32` and `C.uint32_t` is interchangeable and more portable than `C.uint`.